composer.json

```diff
@@ -19,11 +19,11 @@
   "require": {
     "php": ">=7.1",
     "facebook/webdriver": "^1.7.1",
-    "symfony/browser-kit": "^4.3",
-    "symfony/dom-crawler": "^4.3",
-    "symfony/http-client": "^4.3",
+    "symfony/browser-kit": "^4.3|^5.0",
+    "symfony/dom-crawler": "^4.3|^5.0",
+    "symfony/http-client": "^4.3|^5.0",
```

Comment:
It would be consistent to use the same style as below: double pipe+spaces around.